src/Computation/ConstantBitVector.cs

```diff
@@ -17,7 +17,10 @@ private ConstantBitVector(Bits size, IPersistentList<byte> value)
             _value = value;
         }
 
-        public override BigInteger Integer => ToConstantUnsigned().Integer;
+        public override BigInteger GetInteger(Context context)
```

Comment:
I'm guessing these are just degenerate cases that don't need a `context` to compute the integer? My F# brain is hurting because of the unused parameter. Does C# not want you to rename it to `_` to signify that it's unused?

Reply:
C# doesn't mind due to it being on the interface. As long as at least one implementation uses the parameter you don't get a warning or anything.